appengine/Dockerfile

```diff
@@ -1,11 +1,17 @@
 # Base Dockerfile for an App Engine Ruby runtime.
 # Dockerfiles generated by the gcloud tool inherit FROM this image.
-
 FROM gcr.io/google_appengine/base
 
 # Install dependencies for Ruby
 # Also installs dependencies for the following common gems:
-# nokogiri, sqlite3, mysql2, pg
+# nokogiri, sqlite3, mysql2
```

Comment:
Why not list these gems in your table as well:
nokogiri: libxml2-dev
sqlite3: libsqlite3-dev
mysql2: libmysqlclient-dev